Gem/Transfer/ViewsModels/PriceImpactViewModel.swift

```diff
@@ -0,0 +1,91 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+
+import Foundation
+import Primitives
+import Localization
+
+enum PriceImpactValue: Equatable {
```

Comment:
```suggestion
enum PriceImpact: Equatable {
```

or `PriceImpactType`

 You can call it `PriceImpactValue` if it looks like this:

```
struct PriceImpactValue {
     let type: PriceImpactType
     let value: Double
}
```